stories/TeamMembers.tsx

```diff
@@ -0,0 +1,162 @@
+    github: 'https://github.com/alexappleget',
+    linkedin: 'https://www.linkedin.com/in/alex-appleget/',
+    twitter: 'https://x.com/alexlikescoding',
+  },
+  {
+    name: 'Richard Choi',
+    role: 'Developer Relations Engineer',
+    github: 'https://github.com/choir27',
+    linkedin: 'https://www.linkedin.com/in/richard-choir/',
+    twitter: 'https://x.com/choir241',
+  },
+  {
+    name: 'Ryan Furrer',
+    role: 'UX Engineer',
+    github: 'https://github.com/ryandotfurrer',
+    linkedin: 'https://www.linkedin.com/in/ryanfurrer/',
+    twitter: 'https://x.com/ryandotfurrer',
+  },
+  {
+    name: 'Walter Furrer',
+    role: 'Documentation Engineer',
+    github: 'https://github.com/FurrerW',
+    linkedin: 'https://www.linkedin.com/in/furrerw/',
+    twitter: 'https://x.com/furrerw',
+  },
+  {
+    name: 'Corina Murg',
+    role: 'Accessibility Specialist',
+    github: 'https://github.com/CorinaMurg',
+    linkedin: 'https://www.linkedin.com/in/corinamurg/',
+    twitter: 'https://x.com/CorinaMurg',
+  },
+  {
+    name: 'Mai Vang',
+    role: 'Documentation Engineer',
+    github: 'https://github.com/vmaineng',
+    linkedin: 'https://www.linkedin.com/in/mai-vang-swe/',
+    twitter: 'https://x.com/MaiVangSWE',
+  },
+  {
+    name: 'Cody Epstein',
+    role: 'UX Engineer',
+    github: 'https://github.com/kepsteen/',
+    linkedin: 'https://www.linkedin.com/in/cody-epstein/',
+    twitter: '',
+  },
+  {
+    name: 'Michael Larocca',
+    role: 'Documentation Engineer',
+    github: 'https://github.com/MichaelLarocca',
+    linkedin: 'https://www.linkedin.com/in/michaeljudelarocca/',
+    twitter: 'https://x.com/MikeJudeLarocca',
+  },
+  {
+    name: 'Danielle Lindblom',
+    role: 'Frontend Engineer',
+    github: 'https://github.com/Danielle254',
+    linkedin: 'https://www.linkedin.com/in/danielle-lindblom/',
+    twitter: '',
+  },
+  {
+    name: 'Dominick Monaco',
+    role: 'Frontend Engineer',
+    github: 'https://github.com/HoldUpFjord',
+    linkedin: 'https://www.linkedin.com/in/dominick-j-monaco/',
+    twitter: 'https://x.com/DominickJMonaco',
+  },
+];
+
+interface TeamMemberProps {
+  github?: string;
+  linkedin?: string;
+  name: string;
+  role: string;
+  twitter?: string;
+}
+
+const TeamMembers: React.FC = () => {
+  const sortedTeamMembers = [...TeamMembersArray].sort((a, b) => {
+    // Put Shashi Lo first
+    if (a.name === 'Shashi Lo') return -1;
+    if (b.name === 'Shashi Lo') return 1;
+
+    // For the rest, sort by last name
+    const lastNameA = a.name.split(' ').pop() || '';
+    const lastNameB = b.name.split(' ').pop() || '';
+    return lastNameA.localeCompare(lastNameB);
+  });
+
+  return (
+    <div>
+      <h2 className="pt-24 pb-8 text-center sm:text-left">Meet the Team</h2>
+      <div className="team-members-container m-auto grid grid-cols-1 gap-y-8 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4">
+        {sortedTeamMembers.map((member) => (
+          <div
```

Comment:
nit: Make this into a component. Pass the props you need into it.

Reply:
Completed in the latest push.